lib/Mojolicious/Command/nopaste/Service/fpaste.pm

```diff
@@ -0,0 +1,35 @@
+package Mojolicious::Command::nopaste::Service::fpaste;
+use Mojo::Base 'Mojolicious::Command::nopaste::Service';
+
+has description => "Post to fpaste.org\n";
+
+sub paste {
+  my $self = shift;
+  my $lang = $self->language || 'text';
```

Comment:
Ah, indeed, you are right. The default of "perl" comes from here https://github.com/jberger/Mojolicious-Command-nopaste/blob/master/lib/Mojolicious/Command/nopaste/Service.pm#L50 and then if it gets to the individual service without a value, then it should be plain text. Could you please change that one back?